examples/decoding/decoding_csp_timefreq.py

```diff
@@ -46,7 +46,7 @@
 # Extract information from the raw file
 sfreq = raw.info["sfreq"]
 events, _ = events_from_annotations(raw, event_id=dict(T1=2, T2=3))
-raw.pick_types(meg=False, eeg=True, stim=False, eog=False, exclude="bads")
+raw.pick(meg=False, eeg=True, stim=False, eog=False, exclude="bads")
```

Comment:
```suggestion
raw.pick(picks="eeg", exclude="bads")
```

All the other `pick_types` you've changed will need similar treatment